src/constants.ts

```diff
@@ -1,14 +1,11 @@
 export const RLN_ABI = [
-  "function MEMBERSHIP_DEPOSIT() public view returns(uint256)",
-  "function register(uint256 pubkey) external payable",
-  "function withdraw(uint256 secret, uint256 _pubkeyIndex, address payable receiver) external",
-  "event MemberRegistered(uint256 pubkey, uint256 index)",
-  "event MemberWithdrawn(uint256 pubkey, uint256 index)",
+  "function register(uint256[] calldata commitments) external payable",
+  "event MemberRegistered(uint256 idCommitment, uint256 idCommitmentIndex)",
```

Comment:
this abi isn't quite right, we now use an rln registry contract, within which we have references to the storage contracts. you would want to first get the `usingStorageIndex` from the registry contract, then subscribe to `MemberRegistered` from that contract. 
refer to how we do it in nwaku - https://github.com/waku-org/nwaku/blob/2e515a06ed0885a9e2b226a452b36bfdbfef324e/waku/waku_rln_relay/group_manager/on_chain/group_manager.nim#L30-L51 and https://github.com/waku-org/nwaku/blob/2e515a06ed0885a9e2b226a452b36bfdbfef324e/waku/waku_rln_relay/group_manager/on_chain/group_manager.nim#L398-L492

Reply:
Thanks for the pointers